paymentsheet/src/main/java/com/stripe/android/lpmfoundations/paymentmethod/PaymentMethodRegistry.kt

```diff
@@ -0,0 +1,72 @@
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.BancontactDefinition
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.BlikDefinition
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.BoletoDefinition
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.CardDefinition
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.CashAppPayDefinition
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.EpsDefinition
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.FpxDefinition
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.GiroPayDefinition
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.GrabPayDefinition
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.IdealDefinition
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.KlarnaDefinition
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.KonbiniDefinition
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.MobilePayDefinition
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.OxxoDefinition
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.P24Definition
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.PayPalDefinition
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.RevolutPayDefinition
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.SepaDebitDefinition
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.SofortDefinition
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.SwishDefinition
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.UpiDefinition
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.UsBankAccountDefinition
+import com.stripe.android.lpmfoundations.paymentmethod.definitions.ZipDefinition
+
+internal object PaymentMethodRegistry {
+
+    val all: Set<PaymentMethodDefinition> = setOf(
+        AffirmDefinition,
+        AfterpayClearpayDefinition,
+        AlipayDefinition,
+        AlmaDefinition,
+        AmazonPayDefinition,
+        AuBecsDebitDefinition,
+        BacsDebitDefinition,
+        BancontactDefinition,
+        BlikDefinition,
+        BoletoDefinition,
+        CardDefinition,
+        CashAppPayDefinition,
+        EpsDefinition,
+        FpxDefinition,
+        GiroPayDefinition,
+        GrabPayDefinition,
+        IdealDefinition,
+        KlarnaDefinition,
+        KonbiniDefinition,
+        MobilePayDefinition,
+        OxxoDefinition,
+        P24Definition,
+        PayPalDefinition,
+        RevolutPayDefinition,
+        SepaDebitDefinition,
+        SofortDefinition,
+        SwishDefinition,
+        UpiDefinition,
+        UsBankAccountDefinition,
+        ZipDefinition,
+    )
+
+    val map: Map<String, PaymentMethodDefinition> by lazy {
```

Comment:
Nit: Can we come up with a better name here (such as `definitionsByCode`) or completely remove this?

Reply:
Sure